refinery/tool_manager/tests.py

```diff
@@ -1169,6 +1170,33 @@ def test_bad_POST_transaction_rollback(self):
         self.assertIn("ToolDefinition matching query does not exist.",
                       self.post_response.content)
 
+    def test_tool_container_removed_on_deletion(self):
+        with open("{}/visualizations/igv.json".format(TEST_DATA_PATH)) as f:
+            tool_annotation = [json.loads(f.read())]
+
+        with mock.patch(self.mock_vis_annotations_reference,
+                        return_value=tool_annotation):
+            call_command("generate_tool_definitions", visualizations=True)
+
+        td = ToolDefinition.objects.all()[0]
+        post_data = {
+            "tool_definition_uuid": td.uuid,
+            "file_relationships": "['www.coffee.com/cool_file.txt']"
```

Comment:
Should be a IANA-managed Reserved Domain: https://github.com/refinery-platform/refinery-platform/wiki/Copy-Style-Guide#miscellaneous